gadget/validate.go

```diff
@@ -112,6 +112,9 @@ func ruleValidateVolumes(vols map[string]*Volume, model Model, extra *Validation
 		if err := ruleValidateVolume(v, hasModes); err != nil {
 			return fmt.Errorf("invalid volume %q: %v", name, err)
 		}
+		if v.Schema == schemaEMMC && len(vols) > 2 {
+			return fmt.Errorf(`cannot use "schema: emmc" with multiple volumes yet`)
```

Comment:
hm technically the entity with `schema: emmc` is a volume, isn't it?

Reply:
removed this check as this would prevent gadget reuse in the coming volume-assignments